app/layout.tsx

```
@@ -17,6 +17,17 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
+    <head>
+      <script
+        defer
+        data-domain="thedevangel.com"
+        src="https://plausible.byteoski.com/js/script.file-downloads.hash.outbound-links.pageview-props.revenue.tagged-events.js"
+      ></script>
+      <script>
+        window.plausible = window.plausible || function(){" "}
+        {(window.plausible.q = window.plausible.q || []).push(arguments)}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)